src/components/BookItem.js

```diff
@@ -2,30 +2,42 @@
 import React from 'react';
 import { useDispatch } from 'react-redux';
 import { deleteBook } from '../redux/books/books';
+import CircularProgress from '@mui/material/CircularProgress';
 function BookItem(props) {
   const { book } = props;
   const id = book.item_id;
   const dispatch = useDispatch();
 
   return (
```

Comment:
- [ ] Since you didn't make any changes requested by the previous reviewer, I suggest you consider making the changes listed above by the previous review so your project looks like the provided design and is marked as completed

Reply:
I already i have notime to implement suggested changes because its to late now i have done it later its easy because i cover the hard part now i don't want to repeat week for this small thing